docs/rst/notes/notes.rst

```diff
@@ -6,7 +6,40 @@
 Notes
 #####
 
-.. include:: forthcoming_version.rst
+..
+    TODO uncomment when there are forthcoming notes
+    .. include:: forthcoming_version.rst
 
-.. note::
-    This page is under maintenance and will be updated soon.
+Version v0.1.0
+==============
+
+This is the first release of |eddsrecord|.
+
+This release includes several **features** regarding the introspection of DDS data, configuration and user interaction.
+
```

Comment:
I would add the **Recording features** as main features of the application.
e.g. mcap, dyn types, downsampling, activation/deactivation, buffering, etc.